tensorbay/cli/log.py

```diff
@@ -79,11 +81,7 @@ def _implement_log(  # pylint: disable=too-many-arguments
         Printer(dataset_client, revisions, commit_id_to_branches, oneline).generate_commits(),
         max_count,
     )
-    if is_win():
-        for item in commit_generator:
-            click.echo(item)
-    else:
-        click.echo_via_pager(commit_generator)
+    _echo_commit_generator(commit_generator)
```

Comment:
do we still need to add a function `_echo_commit_generator`?
can we revert this change?